pennylane/ops/qubit.py

```diff
@@ -1263,6 +1263,19 @@ def _matrix(cls, *params):
             ]
         )
 
+    @staticmethod
+    def decomposition(phi, theta, omega, wires):
+        decomp_ops = [
+            RZ((phi - omega) / 2, wires=wires[1]),
+            CNOT(wires=wires),
+            RZ(-(phi + omega) / 2, wires=wires[1]),
```

Comment:
Due to the addition and subtraction here, this will _only_ work in tape mode. We have two options:

* Decompose the RZ into two gates to avoid addition/subtraction, however this will increase circuit depth

* Somehow 'turn off' the decomposition outside of tape mode. Or potentially just attempt to `try-except` around the error that is raised if you attempt to add `Variable` objects?